analysis/webservice/algorithms/doms/ResultsStorage.py

```diff
@@ -357,19 +357,18 @@ def __rowToDataEntry(self, row, trim_data=False):
 
         return entry
 
-    def __retrieveStats(self, id):
-        cql = "SELECT num_gridded_matched, num_insitu_matched, time_to_complete FROM doms_execution_stats where execution_id = %s limit 1"
+    def retrieveStats(self, id):
```

Comment:
NOTE: This change was originally made in another PR that hasn't yet been merged. Will rebase once we merged PRs and this should not show up in the diff.